daemon/src/lib.rs

```diff
@@ -64,6 +64,7 @@ pub mod supervisor;
 pub mod taker_cfd;
 pub mod to_sse_event;
 pub mod tokio_ext;
+pub mod transaction_ext;
```

Comment:
Tiny nit: We usually name these `_ext` modules after the crate from where they originate. In this case `bdk` (which re-exports `bitcoin`).

Reply:
Is it a recent convention? I just did the same as in `maia`...

Reply:
removed the file - not needed after all :)